pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1563,6 +1563,9 @@ type PoolPlacementSpec struct {
 	// +kubebuilder:validation:Pattern=`^[a-zA-Z0-9._/-]+$`
 	Name string `json:"name"`
 
+	// Sets given placement as default.
+	IsDefault bool `json:"isDefault"`
```

Comment:
@subhamkrai do you think it's possible to create a validating admission policy spec for the parent that will help ensure only one is marked default?

Reply:
What exactly `by default` do we want to check?  I hope it should be possible to check values for parent spec but I wanted to confirm what exactly we want to check.